lib/extended_button.dart

```diff
@@ -186,24 +190,24 @@ class _ExtendedButtonState extends State<ExtendedButton> {
 
 class SquareBox extends StatefulWidget {
   final IconData icon;
-  final bool opened;
+  final bool opened, isRounded;
   final Function onClick;
   final double closedLeft, openedLeft, closedTop, openedTop, size;
   final Color iconColor, boxColor;
   final Curve curve;
 
-  SquareBox(
-      {this.icon,
-      this.opened,
-      this.onClick,
-      this.closedLeft,
-      this.openedLeft,
-      this.closedTop,
-      this.openedTop,
-      this.iconColor,
-      this.boxColor,
-      this.size,
-      this.curve});
+  SquareBox({this.icon,
+    this.isRounded = false,
```

Comment:
No need to add default value here. The value will never be null when creating `SquareBox`